package.json

```diff
@@ -11,8 +11,7 @@
   },
   "private": true,
   "dependencies": {
-    "@buf/bufbuild_connect-web_open-feature_flagd-dev": "^1.7.2",
```

Comment:
@james-milligan Am I crazy or should this (`@buf/bufbuild_connect-web_open-feature_flagd-dev`) not be here?

I probably accidentally added this dep from your initial PoC. We are doing no buf remote gen here...right?

Reply:
correct, I just looked into it and we are building from the schemas repo in both cases, so we can remove this dependency